doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -123,6 +130,13 @@ Partition Manager
 
 * |no_changes_yet_note|
 
+HID Configurator
+----------------
+
+* Added:
+
+  * HID Configurator now recognizes bootloader variant as a DFU module variant for config channel communication. The new implementation is not backwards compatible.
```

Comment:
> The new implementation is not backwards compatible.

Let's be specific and tell that since new version of script checks for variant it expects firmware to support this request.